src/panels/lovelace/common/validate-condition.ts

```diff
@@ -132,6 +142,62 @@ function checkStateNumericCondition(
   );
 }
 
+function checkLastUpdatedStateCondition(
+  condition: LastUpdatedStateCondition,
+  hass: HomeAssistant
+) {
+  const state_last_changed = (
+    condition.entity ? hass.states[condition.entity] : undefined
+  )?.last_changed;
+  const within = condition.within;
+  const after = condition.after;
+
+  function HaDurationData_to_milliseconds(
```

Comment:
As mentioned: I feel like this function must exist somewhere, but if it doesn't, this is not the right place for it. Any tips for where it should go?